modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -154,6 +136,40 @@ The policy must be in the Stable state to upload the implementation file in a la
 
 To implement the policy so that it is available at runtime, follow the steps in xref:add-policy-implementation-files-to-published-policy-definition.adoc[Add Policy Implementation Files to Published Policy Definition].
 
+== Create a GraphQL API
+A GraphQL API is a specification of a service that defines a schema, which is the collection of object definitions. These objects are used when interacting with the API, either as input data or as part of the API responses. The GraphQL API allows you to query the API instead of accessing an operation that returns a fixed definition of data.
+
+There are two types of schemas that you can upload:
+
+* GraphQL API schema
+* GraphQL Apollo API schema 
```

Comment:
We may want to share what we mean by GraphQL Apollo API schema vs GraphQL API schema, something like:
GraphQL API schema that has been designed with Apollo GraphQL specific directives and structure, also called schema definition language (SDL) at Apollo. 
@IsaacEldridge and @patricio78 for additional inputs if any.

Reply:
Yes, I'd add this as a definition

A GraphQL API schema designed with Apollo GraphQL specific directives and structure, also known as the  schema definition language (SDL).

Reply:
Thanks @IsaacEldridge

Reply:
done